tests/reference/docs/conf.py

```diff
@@ -174,24 +174,41 @@ def download_files_from_gallery(link, folders_in):
     shutil.copyfile(
         'resources/logos/pyfar_logos_fixed_size_pyfar.png', html_logo)
 
+
 # replace imkar hard link to internal link
 with open("_static/header.rst", "rt") as fin:
-    with open("header.rst", "wt") as fout:
-        lines = [line.replace(f'https://{project}.readthedocs.io', project) for line in fin]
-        contains_project = any(project in line for line in lines)
-
-        fout.writelines(lines)
-
-        # add project to the list of projects if not in header
-        if not contains_project:
-            fout.write(f'   {project} <{project}>\n')
-        
-        # count the number of gallery headings
-        count_gallery_headings = np.sum(
-            ['https://pyfar-gallery.readthedocs.io' in line for line in lines])
+    # collect lines in a list
+    lines = [line for line in fin]
+    
+    # find line where pyfar package is mentioned, to determine the start of 
+    # the packages list in the header
+    n_line_pyfar = 0
+    for i, line in enumerate(lines):
+        if 'https://pyfar.readthedocs.io' in line:
+            # remove the 4 lines at the top, which do not include items
```

Comment:
Could you state what they contain? Would be more helpful in case somthing changes and one needs to figure out if the content is relevant or not.